test/e2e_playwright/tests/surveyInviteUser.js

```diff
@@ -0,0 +1,37 @@
+import { DataTestId, getSelector } from '../../../webapp/utils/dataTestId'
+import { gotoSurveyUsersList } from './_navigation'
+
+const expectUser = async ({ user }) => {
+  await expect(page).toHaveText(user)
+}
+
+const expectUsers = ({ users }) => {
+  test(`Check users ${users.length}`, async () => {
+    await page.reload()
+    await Promise.all(users.map(async (user) => expectUser({ user })))
+  })
+}
+
+const inviteUser = () => {
+  test(`Invite user`, async () => {
+    await page.click(getSelector(DataTestId.userList.inviteBtn, 'a'))
+
+    await page.fill(getSelector(DataTestId.userList.email, 'input'), 'testtwo@arena.com')
+
+    await page.click(getSelector(DataTestId.dropdown.toggleBtn(DataTestId.userList.group), 'button'))
+    await page.click(`text="Survey administrators"`)
+
+    await page.click(getSelector(DataTestId.userList.submitBtn, 'button'))
+    await page.waitForTimeout(2500)
```

Comment:
better to wait for response like:

```
await Promise.all([ 
    page.waitForResponse('**/users'), // check the right path though 
    page.click(getSelector(DataTestId.userList.submitBtn, 'button'))
] )
```

**page.waitForTimeout** should be used mainly for debugging https://playwright.dev/docs/api/class-page#pagewaitfortimeouttimeout

Reply:
Changed, thanks